### PR TITLE
Docs: Add a PuppyGraph Integration Guide

### PR DESCRIPTION
**Pull request by FreeMem17**
**PR Checklist**

- [x] A description of the changes is added to the description of this PR.
- [x] If there is a related issue, make sure it is linked to this PR.
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added or modified a feature, documentation in `docs` is updated

**Description of changes**

- Documentation change only
- Add a page for integration of  Unity Catalog and PuppyGraph

### CODE REVIEW

File: docs/integrations/unity-catalog-puppygraph.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
+# Unity Catalog XTable Integration
```
Comment (michelleon) [inline docs/integrations/unity-catalog-puppygraph.md:1]: should be PuppyGraph integration

**Review by michelleon — CHANGES_REQUESTED** [review]
@dennyglee to take a look

other than the one change in the header it looks good to me

File: docs/integrations/unity-catalog-puppygraph.md (hunk 1 above)
Comment (FreeMem17) [reply docs/integrations/unity-catalog-puppygraph.md]: Ah! Good catch.

**Review by dennyglee — APPROVED** [review]
This looks great!  A non-blocking suggestion would be to add some screenshots to PuppyGraph UI to the docs as well.  But this should be done in a different PR.

**Comment by FreeMem17** [comment]
> This looks great! A non-blocking suggestion would be to add some screenshots to PuppyGraph UI to the docs as well. But this should be done in a different PR.

Thank you Denny. I will add them in a separate PR. Is there a general recommendation on image dimension / file size?

**Review by dennyglee — APPROVED** [review]

**Comment by dennyglee** [comment]
> > This looks great! A non-blocking suggestion would be to add some screenshots to PuppyGraph UI to the docs as well. But this should be done in a different PR.
> 
> Thank you @dennyglee. I will add them in a separate PR. Is there a general recommendation on image dimension / file size?

Oh sorry for missing this, GitHub has a max size of 10MB but I would recommend it to be smaller closer to GitHub's [Social Media Preview](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview#).  HTH!

**Pull request merged**